frontend/templates/views/partials/pixi/status-intro.j2

```diff
@@ -15,11 +15,11 @@
       <p>That recommendations will come up Lorem ipsum dolor sit amet, consectetur adipis cing elit, sed do eiusmod tempor incididunt.</p>
     </div>
   </div>
-  <div id="status-intro-banner" class="ap-m-pixi-status-intro-banner">
+  <div id="status-intro-banner-loading" class="ap-m-pixi-status-intro-banner">
     <div class="ap-m-pixi-status-intro-banner-icon"></div>
     <div>
-      <h3>&nbsp;</h3>
-      <p>&nbsp;</p>
+      <h3>Please wait a moment<br>We are currently analyzing your page{% include 'views/partials/pixi/analyzing-dots.j2' %}</h3>
```

Comment:
Enclose translated texts in `{{ _('...') }}`

Reply:
We should take all translated texts out of the pixi/index.md.
That is something we need to change for all components later.